### PR TITLE
Changed package name from twig-drupal to fractal-twig.

### LINKED ISSUE #14: Rename repo to netzstrategen/fractal-twig
And detach from source repo

### PR DESCRIPTION
**Pull request by fabianmarz**
Fixes #14

⚠️ Make sure to adjust the repo name when merging.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,18 +1,18 @@
 {
-  "name": "@netzstrategen/twig-drupal-fractal-adapter",
+  "name": "@netzstrategen/fractal-twig-adapter",
   "version": "2.2.1",
```
Comment (sun) [inline package.json]:
```suggestion
  "version": "2.2.4",
```

File: package.json [hunk 2]
```diff
@@ -1,18 +1,18 @@
 {
-  "name": "@netzstrategen/twig-drupal-fractal-adapter",
+  "name": "@netzstrategen/fractal-twig-adapter",
```
Comment (sun) [inline package.json]:
```suggestion
  "name": "@netzstrategen/fractal-twig",
```

**Review by sun — APPROVED** [review]
Thanks! "fractal-twig" is indeed the correct name.

We should bump the version and also rename the npm package accordingly; I double-checked the suggested name with the other forks listed on https://www.npmjs.com/search?q=fractal-twig

**Comment by fabianmarz** [comment]
Sun, could you please publish to npmjs as well? 🙂 

Edit: Sorry, I missed you did already. Is there a way to mark a repo as stale on npm? If not, we should add a sentence to the README file 🤔

**Pull request merged**